src/components/composites/ImageSlider/ArrowButton.js

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from "@mui/styles";
+import { createTheme, ThemeProvider } from "@mui/system";
+
+const theme = createTheme();
+const useStyles = makeStyles({
+  root: {
```

Comment:
did you write this or is it found code? If the latter, it would be helpful to link where the code is from.